recipes/openssh/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rmdir, export_conandata_patches
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+from os.path import join
+
+required_conan_version = ">=1.54.0"
+
+
+class PackageConan(ConanFile):
+    name = "openssh"
+    description = "The OpenSSH (portable) suite of secure connectivity tools"
+    license = "SSH-OpenSSH"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openssh.com/portable.html"
+    topics = ("security", "cryptography", "login", "keychain", "file-sharing", "ssh")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_libcrypto": [None, "libressl", "openssl"],
+        "with_pam": [None, "openpam"],
+        "with_selinux": [True, False],
+        "with_libedit": [True, False],
+        "with_sandbox": ["auto", "no", "capsicum", "darwin", "rlimit", "seccomp_filter", "systrace", "pledge"]
```

Comment:
```suggestion
        "with_sandbox": [False, "auto", "capsicum", "darwin", "rlimit", "seccomp_filter", "systrace", "pledge"]
```
Would be slightly more consistent with other recipes, imo.

Reply:
IMO, I think that we should have kept the "no" in this case. Variables that are "yes"/"no", ok, let's have True or False as many other recipes, but I don't see the point of mixing boolean and strings in this kind of option. Indeed, it'd avoid the replacement like the comment below https://github.com/conan-io/conan-center-index/pull/19031#discussion_r1538839241